### PR TITLE
fix(): fixing openapigen, codegen and docker_test.go

### PR DESCRIPTION
**Pull request by AniketKul**
fixes https://github.com/ocibuilder/ocibuilder/issues/18 and https://github.com/ocibuilder/ocibuilder/issues/20

### CODE REVIEW

File: hack/update-openapigen.sh [hunk 1]
```diff
@@ -5,11 +5,13 @@ set -o nounset
 set -o pipefail
 
 PROJECT_ROOT=$(cd $(dirname "$0")/.. ; pwd)
-CODEGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/code-generator
+# CODEGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/code-generator
+OPENAPIGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/kube-openapi
 VERSION="v1alpha1"
 
 # Sensor
```
Comment (VaibhavPage) [inline hack/update-openapigen.sh]: You may want to remove the comment #Sensor

File: hack/update-openapigen.sh [hunk 2]
```diff
@@ -5,11 +5,13 @@ set -o nounset
 set -o pipefail
 
 PROJECT_ROOT=$(cd $(dirname "$0")/.. ; pwd)
-CODEGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/code-generator
+# CODEGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/code-generator
```
Comment (VaibhavPage) [inline hack/update-openapigen.sh]: Remove this comment as well

File: hack/update-openapigen.sh (hunk 1 above)
Comment (AniketKul) [reply hack/update-openapigen.sh]: yep. Still in WIP. adding a few other things

File: hack/update-openapigen.sh (hunk 2 above)
Comment (AniketKul) [reply hack/update-openapigen.sh]: fixed it

File: hack/update-openapigen.sh (hunk 1 above)
Comment (AniketKul) [reply hack/update-openapigen.sh]: yes. removed that comment.

**Comment by AniketKul** [comment]
Ready to be merged!

**Comment by AniketKul** [comment]
will rebase this one now

**Comment by AniketKul** [comment]
Alright. those errors are resolved now. ready to be merged!

**Comment by AniketKul** [comment]
@VaibhavPage - All checks have passed. Good to merge!

**Comment by artbegolli** [comment]
Can we add coverage.text to `.gitignore` and not check it in - that one's on me, forgot to add it when i updated the command.

**Comment by artbegolli** [comment]
Also resolve any open discussions that are outstanding

**Comment by AniketKul** [comment]
@artbegolli - sure

**Comment by artbegolli** [comment]
Resolve any final conflicts and I think we're good to go 👍

**Comment by AniketKul** [comment]
Conflicts resolved. Good to merge!

**Pull request merged**